local_sales/app/controllers/products_controller.rb

```diff
@@ -0,0 +1,52 @@
+class ProductsController < ApplicationController
+  before_action :set_product, only: %i[ show edit update destroy ]
+
+  # GET /products
+  def index
+    @products = Product.all
+  end
+
+  # GET /products/1
+  def show
+  end
+
+  # GET /products/new
+  def new
+	  @product = Product.new
+	end
+
+	# GET /products/1/edit
+  def edit
+  end
+
+  # POST /products
+  def create
+    @product = Product.new(product_params)
+	  @product.save
+    redirect_to @product, notice: 'Product was successfully created.'
```

Comment:
I think a validation is missing, I mean, if a product is saved you can redirect and show a message as well as a product is not saved you may determine other actions